ci/cached-builds/gen_gha_matrix_jobs.py

```diff
@@ -198,8 +198,8 @@ def test_select_changed_targets(self):
         changed_files = ["jupyter/datascience/ubi9-python-3.9/Dockerfile"]
 
         leafs = gha_pr_changed_files.filter_out_unchanged(leafs, changed_files)
-        assert set(leafs) == {'amd-jupyter-pytorch-c9s-python-3.9',
-                              'amd-jupyter-tensorflow-c9s-python-3.9',
+        assert set(leafs) == {'rocm-jupyter-pytorch-c9s-python-3.9',
+                              'rocm-jupyter-tensorflow-c9s-python-3.9',
```

Comment:
@atheo89 I suggest we merge it without making the change, and I can do this later, what do you think?

This test does not run in the CI, so we won't spoil anything. And it would be good to start merging the rocm work soon, because the time is running short.

Reply:
When this and the other groundwork gets merged, we can seriously try to decrease image size by removing the unneeded parts of the sdk that are not required to run pytorch/tensorflow. It would be best not to try to release with full sdk, these huge images are too hard to work with.